src/style.css

```diff
@@ -1 +1,64 @@
-/* add styles here */
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-weight: bold;
+}
+
+.page {
+  font-family: Avenir, sans-serif;
+}
+
+.search-bar {
+  display: flex;
+  flex-direction: column;
+  margin-top: 20px;
+}
+
+.search-bar::placeholder {
```

Comment:
The selector `.search-bar::placeholder` is incorrect. The `::placeholder` pseudo-element should be used with an input element, like `.search-bar__input::placeholder`.